tests/test_api.py

```diff
@@ -77,7 +77,7 @@ def test_update_and_check_real_repo(tmpdir):
     repo.head.reset(commit="86a6e6beda8095690414ff7652c15b7ae36e6128", working_tree=True)
     with open(os.path.join(tmpdir, ".cruft.json")) as cruft_file:
         cruft_state = json.load(cruft_file)
-        cruft_state["skip"] = ["cruft/__init__.py", "tests"]
+        cruft_state["skip"] = ["cruft/__init__.py", "tests/*"]
```

Comment:
Is this changed needed? Are top level folders not supported any more?

Reply:
The change is not needed, in fact the behavior would be the same in the 2 cases. I should had not pushed that change, I was debugging some variations locally.